src/main/java/com/backend/response/UpdatePortfolioMetadata.java

```diff
@@ -0,0 +1,9 @@
+package com.backend.response;
+
+import lombok.Data;
+
+@Data
+public class UpdatePortfolioMetadata {
```

Comment:
should follow same naming convention: `UpdatePortfolioMetadataResponse`

Reply:
resolved 😸  [870608e](https://github.com/is442oop/portfolio-analyzer-backend/pull/35/commits/870608ee218040d9255553e09c3974c0e18ed40f)